build-css-testsuites.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env sh
+set -ex
+
+cd "`dirname $0`"
+
+if [ -z $VENV ]; then
+    VENV=tools/_virtualenv
+fi
+
+# Create the virtualenv
+if [ ! -d $VENV ]; then
+    if [ -z $PYTHON ]; then
+        command -v python
+        if [ $? -eq 0 ]; then
+            if [ `python -c 'import sys; print("%i" % (sys.hexversion<0x03000000))'` -eq 1 ]; then
```

Comment:
`python -c 'import sys; print(int(sys.version_info[0] < 3))'` seems easier to understand